client/crud.rst

```diff
@@ -116,12 +116,72 @@ deletion, and the server returned an error indicating that.
 Note that sending an update to a resource after it has been deleted is
 not considered an error and may effectively "undelete" it.
 
+Patching resources
+^^^^^^^^^^^^^^^^^^
+The ``Patch`` interaction on the ``FhirClient`` allows you to send a patch to a FHIR server to update a resource.
+The patch is in the 'FHIRPath Patch' format, which allows you to define a set of operations to apply to a resource using a Paramaters resource in the body of the request.
+Full documentation of FHIRPath Patch can be found `in the FHIR specification <http://hl7.org/fhir/fhirpatch.html>`__.
+
+To use the ``Patch`` interaction, you need to create a ``Parameters`` resource containing the patch operations you want to apply to the resource.
+There are two ways to do this:
+
+1. Create a ``Parameters`` resource and add the patch operations to the ``Parameter`` list manually.
+
+.. code:: csharp
+
+	var patch = new Parameters();
+	patch.Parameter.Add(new Parameters.ParameterComponent()
+	{
+		Name = "OPERATION",
+		Part = new List<Parameters.ParameterComponent>()
+		{
+			new Parameters.ParameterComponent()
+			{
+				Name = "type",
+				Value = new FhirString("replace")
+			},
+			new Parameters.ParameterComponent()
+			{
+				Name = "path",
+				Value = new FhirString("Patient.name[0].given[0]")
+			},
+			new Parameters.ParameterComponent()
+			{
+				Name = "value",
+				Value = new FhirString("John")
+			}
+		}
+	});
+	
+
+2. Create a ``Parameters`` resource and use an extension method of your choice to add the patch operations to the ``Parameter`` list.
+
+.. code:: csharp
+
+	var patch = new Parameters();
+	patch.AddReplacePatchOperation("Patient.name[0].given[0]", new FhirString("John"));
+	
+
+Both methods will result in the same patch operation being sent to the server: It will replace the first given name of the first name of the patient with "John".
```

Comment:
```suggestion
Both methods will result in the same patch operation being sent to the server: it will replace the first given name of the first name of the patient with "John".
```